easybuild/easyconfigs/p/Pysam/Pysam-0.16.0.1-iccifort-2020.1.217.eb

```diff
@@ -0,0 +1,48 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+# 0.9.1.4:
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+# Modified by: Erich Birngruber
+# Gregor Mendel Institute
+# Modified by: Pavel Grochal
+# INUITS
+
+easyblock = 'PythonPackage'
+
+name = 'Pysam'
+version = '0.16.0.1'
+
+homepage = 'https://github.com/pysam-developers/pysam'
+description = """Pysam is a python module for reading and manipulating Samfiles.
+ It's a lightweight wrapper of the samtools C-API. Pysam also includes an interface for tabix."""
+
+toolchain = {'name': 'iccifort', 'version': '2020.1.217'}
+
+source_urls = ['https://github.com/pysam-developers/pysam/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['a61e98e299fa93ba121aab521081282b61bc978bb9052d33b26b38f8fe15573e']
+
+multi_deps = {'Python': ['3.8.2', '2.7.18']}
+
+dependencies = [
+    ('ncurses', '6.2'),
+    ('cURL', '7.69.1'),
+    ('XZ', '5.2.5'),
+]
+
+# required because we're building Python packages using Intel compilers on top of Python built with GCC
+check_ldshared = True
```

Comment:
@Darkless012 It should be OK to remove this (the `PythonPackage` easyblock is now smart enough to auto-enable this when needed, see https://github.com/easybuilders/easybuild-easyblocks/pull/1788)